resources/options_lookup.tsv

```diff
@@ -9772,6 +9775,7 @@ HVAC Heating Efficiency	"MSHP, SEER 29.3, 14 HSPF"	ResStockArguments	heating_sys
+HVAC Heating Efficiency	"MSHP, SEER2 22.7, 10.3 HSPF2, Typical Cold Climate"	ResStockArguments	heating_system_type=none	heating_system_heating_efficiency=0	heating_system_heating_capacity=auto	heating_system_heating_autosizing_limit=auto	heating_system_fraction_heat_load_served=1	heating_system_has_flue_or_chimney=auto	heat_pump_type=mini-split	heat_pump_heating_efficiency_type=HSPF2	heat_pump_heating_efficiency=10.3	heat_pump_cooling_efficiency_type=SEER2	heat_pump_cooling_efficiency=22.7	heat_pump_sizing_methodology=ACCA	heat_pump_sizing_is_duct_limited=false	heat_pump_backup_sizing_methodology=auto	heat_pump_heating_capacity=auto	heat_pump_heating_autosizing_limit=auto	heat_pump_fraction_heat_load_served=1	heat_pump_cooling_capacity=auto	heat_pump_cooling_autosizing_limit=auto	heat_pump_fraction_cool_load_served=1	heat_pump_backup_use_existing_system=false	heat_pump_backup_type=integrated	heat_pump_backup_fuel=electricity	heat_pump_backup_heating_efficiency=1	heat_pump_backup_heating_capacity=auto	heat_pump_backup_heating_autosizing_limit=auto	heat_pump_heating_capacity_retention_fraction=auto	heat_pump_heating_capacity_retention_temp=auto	heat_pump_is_ducted=false	heat_pump_backup_heating_lockout_temp=auto	heat_pump_compressor_lockout_temp=-15	heat_pump_cooling_compressor_type=variable speed	heat_pump_cooling_sensible_heat_fraction=auto	heat_pump_crankcase_heater_watts=auto	geothermal_loop_configuration=none	geothermal_loop_borefield_configuration=auto	geothermal_loop_loop_flow=auto	geothermal_loop_boreholes_count=auto	geothermal_loop_boreholes_length=auto	geothermal_loop_boreholes_spacing=auto	geothermal_loop_boreholes_diameter=auto	geothermal_loop_grout_type=auto	geothermal_loop_pipe_type=auto	geothermal_loop_pipe_diameter=auto
```

Comment:
Looks like we technically don't need this option (since the SDR upgrade yml doesn't reference it), but there's probably no harm in storing it here.